docs/ens_overview.rst

```diff
@@ -0,0 +1,310 @@
+number of subdomains!*
+
+::
+
+    ns.setup_address('supreme.executive.power.derives.from.a.mandate.from.the.masses.jasoncarver.eth')
+
+Wait for the transaction to be mined, then:
+
+::
+
+    assert ns.address('supreme.executive.power.derives.from.a.mandate.from.the.masses.jasoncarver.eth') == \
+        '0x5B2063246F2191f18F2675ceDB8b28102e957458'
+
+Point your address to your name
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Do you want to set up your address so that :meth:`~ens.main.ENS.name` will show the
+name that points to it?
+
+This is like Caller ID. It enables you and others to take an account and
+determine what name points to it. Sometimes this is referred to as
+"reverse" resolution.
+
+::
+
+    ns.setup_name('jasoncarver.eth', '0x5B2063246F2191f18F2675ceDB8b28102e957458')
+
+.. note:: Do not rely on reverse resolution for security.
+
+  Anyone can claim any "caller ID". Only forward resolution implies that
+  the owner of the name gave their stamp of approval.
+
+If you don't supply the address, :meth:`~ens.main.ENS.setup_name` will assume you want the
+address returned by :meth:`~ens.main.ENS.address`.
+
+::
+
+    ns.setup_name('jasoncarver.eth')
+
+If the name doesn't already point to an address, :meth:`~ens.main.ENS.setup_name` will
+call :meth:`~ens.main.ENS.setup_address` for you.
+
+Wait for the transaction to be mined, then:
+
+::
+
+    assert ns.name('0x5B2063246F2191f18F2675ceDB8b28102e957458') == 'jasoncarver.eth'
+
+Auctions for names ending in .eth
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. WARNING::
+  Currently, the registrar functionality is Alpha-quality. Use at your own risk!
+
+  Your funds are your responsibility.
+
+  If you are registering a name, a small misunderstanding can cause you to lose
+  **all** of your deposit. You must `understand ENS auctions
+  <http://docs.ens.domains/en/latest/userguide.html#registering-a-name-with-the-auction-registrar>`_
+  independent of this library.
+
+
+Get auction status
+^^^^^^^^^^^^^^^^^^
+
+Example with domain 'payment.eth':
+
+::
+
+    from ens.registrar import Status
+
+
+    status = ns.registrar.status('payment')
+
+
+    # if you forget to strip out .eth, ens.py will do it for you
+
+    assert ns.registrar.status('payment.eth') == status
+
+
+    # these are the possible statuses
+
+    assert status in (
+      Status.Open,
+      Status.Auctioning,
+      Status.Owned,
+      Status.Forbidden,
+      Status.Revealing,
+      Status.NotYetAvailable
+      )
+
+
+    # if you get the integer status from another source, you can compare it directly
+
+    assert Status.Owned == 2
+
+Start auctions
+^^^^^^^^^^^^^^
+
+::
+
+    # start one auction (which tips people off that you're interested)
+
+    ns.registrar.start('you_saw_him_repressin_me_didnt_ya')
+
+
+    # start many auctions (which provides a bit of cover)
+
+    ns.registrar.start(['exchange', 'tickets', 'payment', 'trading', 'registry'])
```

Comment:
Yeah, that could be helpful (plus it enables a method that would show you if the auction start will end up public, because it's in the dictionary).

> I suspect that this won't get used much

Even if you don't want to use "chaff" auctions, it can be nice to get several that you want started at once (at lower gas cost), like `ns.registrar.start(['pmerriam', 'pipermerriam'])`